el-gr/set-theory-gr.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+category: Algorithms & Data Structures
+name: Set theory
+contributors:
```

Comment:
Oy. Please correct your frontmatter. Add all original contributors to `contributors` and yourself to `translators`.